src/router.js

```diff
@@ -197,6 +209,12 @@ const router = new Router({
       {
         path: 'enfants',
         component: () => import(/* webpackChunkName: "individu" */ './views/Simulation/Enfants.vue'),
+        children: [{
+          name: 'enfants_en_savoir_plus',
+          path: 'en_savoir_plus',
+          props: true,
+          component: () => import(/* webpackChunkName: "individu" */ './components/EnSavoirPlusContent.vue')
+        }]
       },
```

Comment:
La piste des sous-route est effectivement un peu bancale. À y repenser, je ne pensais pas à des sous-routes dans le router mais juste dans l'url.

```suggestion
      {
        path: 'enfants',
        component: () => import(/* webpackChunkName: "individu" */ './views/Simulation/Enfants.vue'),
      },
      {
       path: ':element/en_savoir_plus',
       name: 'en_savoir_plus',
       component: () => import(/* webpackChunkName: "individu" */ './components/EnSavoirPlusContent.vue')
    }
```

devrait éviter les `router-view` est avoir une page dédiée minimaliste.